### PR TITLE
feat: hide/show derived accounts

### PR DESCRIPTION
**Pull request by artsycoder533**
This PR adds the feature of hiding/showing derived accounts.  An `eye` icon is added beside each accounts name.  When a user clicks the `eye` icon, a modal appears asking for confirmation of the account to be hidden.  If the user hits `cancel` the modal closes and nothing happens.  If the user hits `hide account` and object containing the address and the nickname of the corresponding account is added to `wallet.json` in a `hiddenAccounts` array.  This array is used to filter against the array of software and hardware wallets to leave out any accounts that have addresses inside of the `hiddenAccounts` array.  To show a `hidden account` a `Show Accounts` tab has been added to the `Wallet Settings` that allows a user to select which hidden account they want to remove from being hidden.  Once selected, the account is visible again in the `wallet sidebar`.  If no accounts have been hidden a message is displayed saying `There are no hidden accounts to show`.

This PR also cleans up the settings tab sizing to accommodate the new additions as well as move the placement of the edit icon in the wallet sidebar.

*** This issue is a combination of 2 tickets ***
[See Main Linear Ticket #RDX-204 Here for Feature Details](https://linear.app/township/issue/RDX-204/feat-ability-to-hideshow-specific-derived-accounts)
- [See Sub Linear Ticket #RDX-410 Here](https://linear.app/township/issue/RDX-410/hide-derived-accounts)
- [See Sub Linear Ticket #RDX-411 Here](https://linear.app/township/issue/RDX-411/show-derived-accounts-that-have-been-hidden)

### CODE REVIEW

File: src/components/layout/WalletSidebarAccounts.vue [hunk 1]
```diff
@@ -176,7 +177,22 @@ const WalletSidebarAccounts = defineComponent({
...
+        const isLocalAccount = account.signingKey.isLocalHDSigningKey
+        // flatten array of objects
+        const newArr = hiddenAccounts.value.flatMap(acct => Object.values(acct))
+        const isHidden = newArr.includes(account.address.toString())
+        return isLocalAccount && !isHidden
+      })
+    })
+
+    const nonHiddenHardwareDevices: ComputedRef<HardwareDevice[]> = computed(() => {
+      console.log('here')
```
Comment (staylorwr) [inline src/components/layout/WalletSidebarAccounts.vue:189]: Can we clean this log up?

File: src/composables/useWallet.ts [hunk 2]
```diff
@@ -408,8 +413,32 @@ const handleDecimalType = (newDecimalType: AmountFormats) => {
...
+// get hiddenAccounts
+const fetchHiddenAccountsFromElectron = async () => {
+  hiddenAccounts.value = await getHiddenAccounts()
+}
+
+fetchHiddenAccountsFromElectron()
+
+// handleHiddenAccounts
+const handleHiddenAccounts = async (newAcctToHide: string, newAcctNickname: string) => {
+  console.log(newAcctToHide, newAcctNickname)
```
Comment (staylorwr) [inline src/composables/useWallet.ts:427]: Another log snuck through :)

File: src/composables/useWallet.ts [hunk 3]
```diff
@@ -722,6 +757,12 @@ const decryptMessage = async (tx: ExecutedTransaction): Promise<string> => {
 
 const setHideAccountModal = (val: boolean) => { showHideAccountModal.value = val }
 
+const setActiveAccountAddress = (addr: string, acctNickName: string) => {
+  // console.log(addr, acctNickName)
```
Comment (staylorwr) [inline src/composables/useWallet.ts:761]: log log log

**Review by staylorwr — APPROVED** [review]
Code & UI Look great.  Just a few logs to clean up

File: src/components/layout/WalletSidebarAccounts.vue (hunk 1 above)
Comment (artsycoder533) [reply src/components/layout/WalletSidebarAccounts.vue]: @staylorwr Removed them!

**Pull request merged**